main.go

```diff
@@ -172,6 +173,16 @@ func _main(ctx context.Context) error {
 			DBName:       cfg.PGDBName,
 			QueryTimeout: cfg.PGQueryTimeout,
 		}
+		// Replace {netname} with curnet
```

Comment:
For consistent style, please put a newline before the comment.  Not all blocks are separated by newlines of course, but chaining too many together gets confusing.  Plus it's almost always the case that if a chunk of code is commented like this, we will precede it with a newline.